src/Avalonia.Themes.Default/DefaultTheme.xaml

```diff
@@ -1,70 +1,105 @@
+        <ResourceInclude Source="avares://Avalonia.Themes.Default/Controls/OverlayPopupHost.xaml" />
+        <ResourceInclude Source="avares://Avalonia.Themes.Default/Controls/WindowNotificationManager.xaml" />
+        <ResourceInclude Source="avares://Avalonia.Themes.Default/Controls/EmbeddableControlRoot.xaml" />
+        <ResourceInclude Source="avares://Avalonia.Themes.Default/Controls/TransitioningContentControl.xaml" />
+        <ResourceInclude Source="avares://Avalonia.Themes.Default/Controls/TreeView.xaml" />
+        <ResourceInclude Source="avares://Avalonia.Themes.Default/Controls/TreeViewItem.xaml" />
+        <ResourceInclude Source="avares://Avalonia.Themes.Default/Controls/ProgressBar.xaml" />
+        <ResourceInclude Source="avares://Avalonia.Themes.Default/Controls/Separator.xaml" />
+        <ResourceInclude Source="avares://Avalonia.Themes.Default/Controls/Menu.xaml" />
+        <ResourceInclude Source="avares://Avalonia.Themes.Default/Controls/MenuItem.xaml" />
+        <ResourceInclude Source="avares://Avalonia.Themes.Default/Controls/ContextMenu.xaml" />
+        <ResourceInclude Source="avares://Avalonia.Themes.Default/Controls/CalendarButton.xaml" />
+        <ResourceInclude Source="avares://Avalonia.Themes.Default/Controls/CalendarDayButton.xaml" />
+        <ResourceInclude Source="avares://Avalonia.Themes.Default/Controls/CalendarItem.xaml" />
+        <ResourceInclude Source="avares://Avalonia.Themes.Default/Controls/Calendar.xaml" />
+        <ResourceInclude Source="avares://Avalonia.Themes.Default/Controls/CalendarDatePicker.xaml" />
+        <ResourceInclude Source="avares://Avalonia.Themes.Default/Controls/Slider.xaml" />
+        <ResourceInclude Source="avares://Avalonia.Themes.Default/Controls/NotificationCard.xaml" />
+        <ResourceInclude Source="avares://Avalonia.Themes.Default/Controls/NativeMenuBar.xaml" />
+        <ResourceInclude Source="avares://Avalonia.Themes.Default/Controls/NumericUpDown.xaml" />
+        <ResourceInclude Source="avares://Avalonia.Themes.Default/Controls/DateTimePickerShared.xaml" />
+        <ResourceInclude Source="avares://Avalonia.Themes.Default/Controls/DatePicker.xaml" />
+        <ResourceInclude Source="avares://Avalonia.Themes.Default/Controls/TimePicker.xaml" />
+        <ResourceInclude Source="avares://Avalonia.Themes.Default/Controls/SplitView.xaml" />
+        <ResourceInclude Source="avares://Avalonia.Themes.Default/Controls/ManagedFileChooser.xaml" />
+        <ResourceInclude Source="avares://Avalonia.Themes.Default/Controls/SplitButton.xaml" />
+        <ResourceInclude Source="avares://Avalonia.Themes.Default/Controls/RichTextBlock.xaml" />
+      </ResourceDictionary.MergedDictionaries>
+    </ResourceDictionary>
+  </Styles.Resources>
+
+  <StyleInclude Source="avares://Avalonia.Themes.Default/Controls/FocusAdorner.xaml" />
+  <StyleInclude Source="avares://Avalonia.Themes.Default/Controls/UserControl.xaml" />
+
+  <Style Selector="Window">
```

Comment:
@grokys for some reason dev tools doesn't work when I replace Window style with a control theme. Can you take a look?